_includes/components/department-budgets/DeptSectionHead/index.html

```diff
@@ -0,0 +1,3 @@
+<div class="DeptSectionHead DeptSectionHead--{{ include.color }}" id="{{ include.id }}">
```

Comment:
Resist the temptation to create components unnecessarily, it's not saving any effort. This can quite easily be in-lined into the page itself since it's a single tag.

Would an H tag make more sense than a DIV?

Reply:
Right will do.